spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java

```diff
@@ -101,6 +101,31 @@
 	 */
 	private String protocol = "TLS";
 
+	/**
+	 * Disable SSL compression.
+	 */
+	private Boolean disableCompression;
+
+	/**
+	 * Honor SSL cipher suite order.
+	 */
+	private String honorCipherOrder;
```

Comment:
While Tomcat uses a `String` for this it's coerced to a `boolean`. I think it'd be better if we made this more strongly typed and used a `Boolean`.